server/repositories/__tests__/hubMenuRepository.spec.js

```diff
@@ -67,4 +68,34 @@ describe('hubMenuRepository', () => {
       expect(result).toStrictEqual(expected);
     });
   });
+
+  describe('tagsMenu', () => {
+    it('it returns a tag menu', async () => {
+      const httpClient = {
+        get: jest.fn().mockReturnValue(vocabularyResponse),
+      };
+
+      const repository = hubMenuRepository(httpClient);
+      const expected = [
+        {
+          id: 729,
+          linkText: 'Resettlement',
+          href: '/tags/729',
+          description: '<p>Challenges faced when getting out.</p> ',
+        },
+        {
+          id: 724,
+          linkText: 'Success stories',
+          href: '/tags/724',
+          description: '<p>Real stories from real people.</p> ',
+        },
+      ];
+
+      const result = await repository.tagsMenu({
+        prisonId: 2,
+      });
+
+      expect(result).toStrictEqual(expected);
+    });
+  });
```

Comment:
I'm probably missing something, but does this test assert that the request to the API includes the `prisonId`?

Reply:
I suppose we could assert on the mock that it's been called and with the `prisonId`?

We're not actually calling the API here, we're calling `httpClient.get()` which builds the query and sends it off.

Reply:
I could amend all the tests in this since none of them do that either. Or it can get looked at as tech debt.

Reply:
That's an interesting one. You're right, asserting the mocked `httpClient` call includes the `prisonId` would give us a test which would have failed before the change in this PR 👍.

As for all the tests, I guess if we have at least one including the `prisonId` we've got coverage. It's not clear cut, but I tend to prefer to test what mocks have been called with as that's the boundary we're introducing for the test... though sometimes that can be more effort than it's worth 🤷 .